justfile

```diff
@@ -89,8 +82,7 @@ release version:
 
 # extract tag from pyproject.toml
 get-tag-version:
-    #!/usr/bin/env bash
-    uvx --from=toml-cli toml get --toml-path=pyproject.toml "project.version"
+    @uvx --from=toml-cli toml get --toml-path=pyproject.toml "project.version"
```

Comment:
what does '@' do in a justfile?

Reply:
just suppresses the command to stdout

#### with `@`
```just
hello:
  @echo "hello world"
```

```
hello world
```

#### without

```just
hello:
  echo "hello world"
```

```
echo "hello world!"
hello world
```